CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Allow flatcar alpha releases for machine pools.
```

Comment:
generically, with this change we allow to change flatcar to any (newer) release, not just alpha. Shall we rephrase the changelog to explain that?

Reply:
Changed it slightly 👍🏻